### DISCUSSION TITLE
PlanetMinecraft (UserStyle)

### DISCUSSION THREAD

**Discussion by Nyameliaaaa** [catppuccin/catppuccin#2139] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://userstyles.world/style/9060/classic-planetminecraft

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closed as accepted here: https://github.com/catppuccin/userstyles/pull/200
